Pod/Classes/BONChain.m

```diff
@@ -441,4 +441,14 @@ - (BONChainColor)textColor
     return self.color;
 }
 
+- (void)appendLink:(id<BONTextable>)link separator:(NSString *)separator
+{
+    if (separator.length > 0) {
```

Comment:
#pp I think you can just forward this to `appendLink:separatorTextable`, and wrap `separator` in a `BONChain`. Might be problematic because you modified the tests instead of adding new tests, so they don't test this method any more. Not sure if it's worth keeping the old and new tests, though ¯\\\_(ツ)\_/¯